samples/csharp_dotnetcore/48.qnamaker-active-learning-bot/README.md

```diff
@@ -10,7 +10,7 @@ In this sample, we demonstrate how to use the Active Learning to generate sugges
 - Follow instructions [here](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/set-up-qnamaker-service-azure)
 to create a QnA Maker service.
 - Follow instructions [here](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/quickstarts/create-publish-knowledge-base) to
-import and publish the [ActiveLearningSampleFAQ.tsv](CognitiveModels/ActiveLearningSampleFAQ.tsv) to your newly created QnA Maker service.
```

Comment:
why delete this ActiveLearningSampleFAQ.tsv? i run this project in my local, but i can't understand well without a kb, cuz, i can't understand this `GetLowScoreVariation` in DialogHelper.cs, could you please help to clarify? or give an example?  will appreciate

Reply:
The file was not included by the original creator of the sample, but it was referenced in the Readme.md If the file were present in the sample, then we would leave the text in the Readme.

Reply:
Thanks for your response. i have another question, but seems i should comment it in` DialogHelper.cs`